Fluent/Simulation.ipynb

```diff
@@ -5,198 +5,94 @@
+    "**4. Setup Solution methods:**\n",
+    "In this step, it is highly recommended to use the default options and settings, unless based on physics of the problem the user is aware of any specific choices. Upon non-smooth convergence and potential divergence of the CFD simulation user can modify and examine various solution methods. To modify the solution methods and controls use the following commands respectively:\n",
     "\n",
-    "```\n",
-    "from IPython.display import YouTubeVideo\n",
-    "YouTubeVideo(\"kyAJiOXQIq4\")\n",
-    "```"
+    "solve/set/discretization-schem\n",
+    "\n",
+    "solve/set/under-relaxation\n",
+    "\n",
+    "Now all boundary conditions and settings for the CFD simulation are defined. User can **initialize** the solution through an educated guess to start the iteration process: /solve/initialize/compute-defaults/velocity-inlet Solution initialization would incept the flow field variables, such as velocity and pressure, based on the defined values by user. For the current problem the CFD domain is recommended to be initialized using values of velocity and pressure at the pipe's inlet.\n",
+    "\n",
+    "For the unsteady case input is required for the time step size and number of time steps. Other settings may be left to default. The alternating vortices in transient flow require a time step that is low enough to account for the rapid changes. Users can go through a trial and error process to find which time step size is appropriate for their particular problem. A helpful way to analyze this process is to visualize it by using animation playback sequence. Before running the calculation, user creates a solution animation sequence via: solve/calculation-activities/solution-animations. The animations can then be played back during post processing. If the time step is too large the solution will skip over parts of the simulation, which can be seen in the animation by a non-smooth transition between frames. If the time step is too small the solution will have more data than is necessary for parts of the simulation, which takes up time to compute, and is shown in the animation by the frames changing by very little. For this problem, a time step size of 0.01 [s] is sufficient: solve/run-calculation/time-step-size. The number of time steps for this case should be at least 1000: solve/run-calculation/number-of-time-steps to allow for convergence to occur. As the rational speed increases, it is advised to increase the number of time steps as it takes more time to reach a steady like solution. Also to observe the disappearance of the vortices it takes more time. Suggested value is at least 5000 steps. Also chose the number of iterations per time step wisely. Provide enough time for the residuals to reach a very low value.\n",
```

Comment:
Review and review the wording of this paragraph. The first half has been revised. However, the content of the second half, specially toward the end is written poorly. The message of the content was so wage that it was not understood to implement direct changes.